debian/po/templates.pot

```diff
@@ -0,0 +1,42 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the raspberrypi-sys-mods package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+#
+#, fuzzy
+msgid ""
+msgstr ""
+"Project-Id-Version: raspberrypi-sys-mods\n"
+"Report-Msgid-Bugs-To: raspberrypi-sys-mods@packages.debian.org\n"
+"POT-Creation-Date: 2021-02-07 12:06-0500\n"
+"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"Language-Team: LANGUAGE <LL@li.org>\n"
+"Language: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=CHARSET\n"
+"Content-Transfer-Encoding: 8bit\n"
+
+#. Type: select
+#. Description
+#: ../templates:1001
+msgid "Allow adding third party repositories?"
+msgstr ""
+
+#. Type: select
+#. Description
+#: ../templates:1001
+msgid ""
+"This package may add third party repositories and keys when upgrading. "
```

Comment:
Does it make sense to allow this generally? I think a 3rd party repository should not added at all by an APT package, unless you explicitly choose to install the software that is provided by this 3rd party repository.

Since the decision about the VS Code topic has been done already, at least a debconf choice should not grant now to add unlimited further repos. Instead this should be a very rare (or the only) exception and if for some reason the question is to add another 3rd party repo, this should require an own debconf question for this specific new repository.

Reply:
This is just a merge of the changes that @XECDesign has already pull into the HEAD to prevent conflicts. This PR, 51, does provide user with choice as explained above:
https://github.com/RPi-Distro/raspberrypi-sys-mods/pull/51#discussion_r571704423